resources/META-INF/plugin_template.xml

```diff
@@ -211,10 +211,19 @@
     <extensionPoint name="gradleSyncProvider" interface="io.flutter.android.GradleSyncProvider"/>
   </extensionPoints>
 
+  <extensionPoints>
+    <extensionPoint name="colorPickerProvider" interface="io.flutter.editor.ColorPickerProvider"/>
```

Comment:
The comments I made on plugin.xml actually belong here.

Reply:
done.